spec/workers/legacy_sync_worker.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
```

Comment:
I see there is a testing library for sidekiq. https://github.com/mperham/sidekiq/wiki/Testing
Would be good to hear from the others whether they feel its worth adding tests for all workers.

Reply:
It would be helpful, but as long as the lib code works it should be good to go. Workers should really only ever call code inside the app

Reply:
yeah, I'd test the logic that the worker runs, rather than test the worker itself, if that makes sense.